src/annotator/integrations/vitalsource.js

```diff
@@ -184,6 +216,14 @@ export class VitalSourceContentIntegration {
       });
     }
 
+    // Install scrolling workaround for PDFs. We do this in the content frame
+    // so that it works whether Hypothesis is loaded directly into the content
+    // frame or injected by VitalSourceInjector from the parent frame.
```

Comment:
What would the alternative be? (The way this is worded it sounds like the "content frame" was chosen over...something else?).

Reply:
An alternative place to put this code would be in the `VitalSourceInjector` class, so that it runs in the parent frame to the one where `VitalSourceContentIntegration` runs.